### PR TITLE
Openshift logging & metrics bzs

### PR DESCRIPTION
**Pull request by ewolinetz**
Addresses:
  https://bugzilla.redhat.com/show_bug.cgi?id=1430627
  https://bugzilla.redhat.com/show_bug.cgi?id=1430626
  https://bugzilla.redhat.com/show_bug.cgi?id=1430628

### CODE REVIEW

**Comment by ewolinetz** [comment]
This likely needs a cherry-pick into master too...

File: roles/openshift_logging/tasks/generate_pvcs.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ---
+- debug: var=es_pvc_size
+- debug: var=es_cluster_size
+- debug: var=es_dc_names
```
Comment (sdodson) [inline roles/openshift_logging/tasks/generate_pvcs.yaml]: intentionally left in?

File: inventory/byo/hosts.origin.example [hunk 2]
```diff
@@ -530,8 +530,8 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # list of options please see roles/openshift_logging/README.md
 #
 # Configure loggingPublicURL in the master config for aggregate logging, defaults
-# to https://kibana.{{ openshift_master_default_subdomain }}
-#openshift_hosted_logging_hostname=https://kibana.example.com
+# to kibana.{{ openshift.common.dns_domain }}
```
Comment (sdodson) [inline inventory/byo/hosts.origin.example]: hmm, dns_domain is usually 'cluster.local' so internal dns not public dns. I think you want `kibana.{{ openshift_master_default_subdomain }}`

Comment (sdodson) [inline inventory/byo/hosts.origin.example]: About this we probably need to fix https://github.com/openshift/openshift-ansible/blame/master/roles/openshift_logging/defaults/main.yml#L25 too

Comment (sdodson) [inline inventory/byo/hosts.origin.example]: https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_hosted.yml#L23 is the default logic used elsewhere

File: roles/openshift_logging/tasks/generate_pvcs.yaml (hunk 1 above)
Comment (ewolinetz) [inline roles/openshift_logging/tasks/generate_pvcs.yaml]: nope, thanks for catching that...

File: inventory/byo/hosts.origin.example (hunk 2 above)
Comment (ewolinetz) [inline inventory/byo/hosts.origin.example]: that is what we use in the openshift_logging role though. 
from defaults/main.yaml
`openshift_logging_kibana_hostname: "{{ openshift_hosted_logging_hostname | default('kibana.{{openshift.common.dns_domain}}') }}"`

Its in the case where they are using the playbooks/byo/openshift-cluster/openshift-logging.yaml playbook. That may not be set

**Comment by ewolinetz** [comment]
@sdodson should we be failing/setting a default value of `openshift_master_default_subdomain` then within the `openshift_logging` role?

**Pull request merged**